src/api/profile.ts

```diff
@@ -48,3 +48,17 @@ export const makeUpdateUserGridLocation = ({
       firestore.doc(doc).set(newData);
     });
 };
+
+export interface makeUpdateUserOnlineStatusProps {
+  userUid: string;
+}
+
+export const makeUpdateUserOnlineStatus = ({
+  userUid,
+}: makeUpdateUserOnlineStatusProps) => (status: string) => {
+  const newData = {
+    "data.status": status,
+  };
+
+  firebase.firestore().collection("users").doc(userUid).update(newData);
```

Comment:
Add bugsnag to an error catcher + return firease call.

As per https://github.com/sparkletown/sparkle/blob/feature/new-auditorium/src/api/profile.ts